### PR TITLE
Jarvis backports

### PR DESCRIPTION
**Pull request by Jalle19**
I adapted travis.yml to build against the Kodi Jarvis branch, let's see how it goes.

### CODE REVIEW

**Comment by MartijnKaijser** [comment]
seems to work.

**Comment by Jalle19** [comment]
@ksooo good to go?

File: src/HTSPDemuxer.cpp [hunk 1]
```diff
@@ -159,21 +162,28 @@ bool CHTSPDemuxer::Seek
   if (!m_subscription.IsActive())
     return false;
 
-  if (!m_subscription.SendSeek(time))
+  m_seeking = true;
+  if (!m_subscription.SendSeek(time)) {
```
Comment (ksooo) [inline src/HTSPDemuxer.cpp]: Can you do me a favour and put the curly bracket one line down? I hate this coding style.... Just asking.

File: src/HTSPDemuxer.cpp [hunk 2]
```diff
@@ -184,6 +194,10 @@ void CHTSPDemuxer::Speed ( int speed )
   CLockObject lock(m_conn.Mutex());
   if (!m_subscription.IsActive())
     return;
+  if (speed != m_subscription.GetSpeed() && (speed < 0 || speed >= 4000)) {
```
Comment (ksooo) [inline src/HTSPDemuxer.cpp]: Hmm.

**Comment by ksooo** [comment]
Good to go. 

(In case you've seen my - meanwhile deleted comments - re. coding style - forget about it. This is just a backport, not worth the effort. ;-)

**Pull request merged**